docs/benchmarks/benchmarks.rst

```diff
@@ -62,6 +63,14 @@ This creates designs of a variety of sizes and computes the error on unseen data
 function. It compares the accuracy of the sequential design to the Latin Hypercube for both the
 predictions and uncertainties.
 
+Pivoted Cholesky Benchmark
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A benchmark shows how a duplicated input can degrade emulator performance
```

Comment:
"A benchmark shows how ..." => "A benchmark that shows how ..." ?

Reply:
Also good catch, as with the previous I'll fix with another set of changes I have ready to go.